test/test.js

```diff
@@ -0,0 +1,118 @@
+var assert = require('assert');
```

Comment:
You're manipulating the real data. Think of a DB usage in a project. You won't like your function to manipulate the real data in the tests. The functions you test should be independent and testable

Reply:
What about the Routing tests?

Reply:
Don't forget the it should standard!